purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesException.kt

```diff
@@ -9,5 +9,9 @@ open class PurchasesException(val error: PurchasesError) : Exception() {
         get() = error.underlyingErrorMessage
 
     override val message: String
-        get() = error.message
+        get() = error.message + if (error.underlyingErrorMessage != null) {
+            " Underlying error: ${error.underlyingErrorMessage}"
+        } else {
+            ""
+        }
```

Comment:
hmm isn't the `underlyingErrorMessage` most of the time something not that user friendly? If someone is displaying just the message, this change will show the whole thing. I beleive for request errors, the underlying error message is particularly dev-focused.

For example the other day building an error view for the Customer Center, in my first iteration I added the underlying message in an initial iteration, but then I noticed it was very developer focused and decided to remove it from the UI. I kept the message though ("Error performing request"). I believe the underlying error message said something about can't decode "<html" tag but I don't recall exactly.